### ISSUE TITLE
Outbound connectivity from scaling lightning network

### ISSUE THREAD

**Issue by Jhoyola** [scaling-lightning/scaling-lightning#20]
For local development (and maybe some other use cases too) would be good to have a possibility to make a connection from the scaling lightning network to an external host. For example: I have some service that I'm developing exposing an API, running on my localhost. I also have scaling lightning network running on my computer. Some instance in scaling lightning would use the API.

A solution could be to create a new "instance type" (not sure how to call it) to scaling lightning that would be the proxy/tunnel to the localhost (or some other host).

Some possibilities:

- https://github.com/we-dcode/kubetunnel
- https://webapp.io/blog/container-tcp-tunnel/
- https://github.com/omrikiei/ktunnel
- https://www.perdian.de/blog/2022/02/21/setting-up-a-wireguard-vpn-using-kubernetes/

**Comment by maxwedwards** [comment]
Agree, @Jhoyola this would be really useful. Thanks for the issue.
